db/fixtures/miq_event_definitions.csv

```diff
@@ -193,7 +193,6 @@ containernode_compliance_failed,Container Node Compliance Failed,Default,complia
 
 containergroup_deadlineexceeded,Pod Deadline Exceeded,Default,container_operations
 containergroup_failedscheduling,Pod Failed Scheduling,Default,container_operations
-containergroup_failedsync,Pod Failed Sync,Default,container_operations
```

Comment:
@simon3z this looks like it is related to the policy_event kicked off by https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/System/Event/EmsEvent/Kubernetes.class/pod_failedsync.yaml#L14 but doesn't appear to have anything to do with the EventCatcher

I'd say lets leave this for now (it isn't hurting anything) and just remove FailedSync from the `ENABLED_EVENTS`.